modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -12,7 +12,7 @@ The way to create an asset depends on its type. The type is set when the asset i
 * For OAS, RAML, RAML fragments, AsyncAPI, HTTP, WSDL, rulesets, and custom assets, create the asset directly using the Exchange *Publish new asset* menu.
 * For an example or template, create each using the Mavenize feature in Anypoint Studio, and publish each to Exchange.
 * For a connector, policy, example, or template, see xref:to-publish-assets-maven.adoc[Publish and Deploy Exchange Assets Using Maven].
-* You can also publish a RAML, or OAS or RAML fragments using Design Center and publish the API to Exchange.
+* You can also publish a RAML, OAS components, and JSON and RAML fragments using Design Center and publish the API to Exchange.
```

Comment:
What do you think if we talk about asset types instead of different technologies? I think it can avoid misunderstandings and it is more concise

```suggestion
* You can also publish REST-APIs and API Spec Fragments using Design Center and publish the API to Exchange.
```

Reply:
yes this makes sense to me, if we explain the granularity within REST-APIs below

Reply:
fixed